### DISCUSSION TITLE
[Q&A] Mac-address computer not inventoried

### DISCUSSION THREAD

**Discussion by marioalpha** [glpi-project/glpi-agent#784] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Good morning,

I am distributing the new Agent 1.11
The Mac-Address field is no longer present in "Network Port" but is present only in "Component"
This behavior is also removing information on "Network Devices" on the Switch doors I no longer have your connection to the computer
What is going on?

Glpi 10.0.16
Glpi Agent 1.11
Apache 2.4.29
PHP 8.2.4

**Comment by g-bougard** [comment]
Hi @marioalpha 

you're not clear enough: what kind of device are you speaking about ? Computer or network device ? Is this new in the sens it worked before with another version ? If yes, check this still works with the older version and so share the involved inventory before & after.

Have you any inventory to share showing something is wrong ?

**Comment by marioalpha** [comment]
I was analyzing the behavior of the new agent and I noticed that the Switch inventory are losing connections to computer objects.

What I wrote is incorrect, I had an oversight, sorry for having incorrectly opened the post.

Computer_Network ports
![Computer_Network ports](https://github.com/user-attachments/assets/17481649-30e4-4111-a0fd-5db3260f3a93)

Computer_Network_Historical
![Computer_Network_Historical](https://github.com/user-attachments/assets/a407ffd9-e3e2-42d9-98ed-fce66108d53a)

Network_device_port
![Network_device_port](https://github.com/user-attachments/assets/f994e20e-2847-49a8-8951-b9672163c6a1)
